### ISSUE TITLE
Cookie Policy: No document option?

### ISSUE THREAD

**Issue by paaljoachim** [Really-Simple-Plugins/complianz-gdpr#329]
I see this Documents screen.

![Screenshot 2021-12-31 at 10 52 14](https://user-images.githubusercontent.com/5323259/147816498-94a51730-6a0d-465f-b3da-b17e66c97b71.jpg)

Thinking basic I would likely not create a Cookie Policy page but instead have a Privacy Statement. (For my case a GDPR page to use in Norway.)

Perhaps I am mixing Cookie Policy with Privacy Statement. Currently we just have a GDPR page telling the user how we use cookies and how their privacy is taken care of. 

Bottom line is that I become a bit confused. As I do not want to create a Cookie Policy but instead create a Privacy Statement that will cover the various needs.

**Comment by paaljoachim** [comment]
Currently this means I will select: "Generated by Complianz" and remove the page from the menu, and instead add the Privacy Statement (GDPR) page to the menu.

**Comment by Paapst** [comment]
A cookiepolicy (aka dnsmpi/Optout preferences) and a privacy statement have different goals. This is why Complianz, but also the other "bigger" players such as OneTrust, has always divided those into separate pages. 

There are some regions that mandate that information about cookies and/or tracking/ or opt-out possibilities is explicitly mentioned in a clear and prominent notice, and that other information should be listed on a page called "Privacy". 
The GDPR and the ePrivacy Directive might not mandate that (yet), but also the GDPR requires that any information or communication relating to the processing of personal data is **easily accessible** and is available in clear and plain language. If you would provide all that information about the cookies, the use of local storage, or other technologies in the same document as to where you describe all the methods and purposes of the data processing of your entire organisation, you will make it potentially difficult for your visitors to understand for what purposes they are actually giving their consent. And especially if you combine it with a so-called tcf banner that contains a list of IAB vendors.  

Having two different pages by default also makes it easier to update them should a region decide to mandate two separate documents/pages/notices, or -as is now the case in some US states- should it become neccesary to create a Opt-out preferences page.

**Comment by rlankhorst** [comment]
In this case I would recommend to use the option to link to a custom page, in the cookie policy question. This will put your privacy policy on your cookie banner, with the page title you use there. This should be sufficient for your purposes I think.

**Comment by paaljoachim** [comment]
Thank you for the feedback @Paapst and @rlankhorst 

Can we also for 7.0 basic/intermedia/advanced steps also think about how tiny/medium/big companies might have different privacy needs? As some companies have very few privacy concerns while more complex web sites will likely have multiple privacy concerns for various ways they use user information.

**Comment by rlankhorst** [comment]
Hi @paaljoachim, I refer to our lawyer @Paapst here as I’m not the legal specialist, but I would say that the requirements regarding privacy are not really different for different company sizes. Of course there may be other usage on a smaller site, but that is what the wizard is for: to determine how your site is used.
